webapp/src/features/event/basic/EventDetails.tsx

```diff
@@ -1,14 +1,40 @@
 import { useEventOperations } from "../outletContext";
 import EventBasicDetailsForm from "./EventBasicDetailsForm";
 import "react-datepicker/dist/react-datepicker.css";
+import { Button, Card, CardBody, Form } from "react-bootstrap";
+import { getSignupsPostedMessage } from "../../../util/messageWriters";
 
 const EventDetails = () => {
 
     const [eventScratchpad, proposeEventChange] = useEventOperations();
+    const message = getSignupsPostedMessage(eventScratchpad);
 
     return <>
-        <h2 className="display-6">Event Setup</h2>
-        <EventBasicDetailsForm event={eventScratchpad} onEventChange={proposeEventChange} />
+        <Card>
```

Comment:
React structure-wise, would it be better to move the "signup sheet" section to its own component here, or do you think this is probably fine as is?